server/ansible/roles/pbench-server-activate-create-crontab/tasks/main.yml

```diff
@@ -1,5 +1,7 @@
 ---
 - name: create crontab
+  become: yes
+  become_user: pbench
```

Comment:
NOTE: I'm using this as the Ansible equivalent of `su pbench -c`. I ought to have been able to remove the `PYTHONPATH` definition from the `environment`, too, but I didn't think of that at the time.